plugins/tutor-contrib-blockstore-filesystem/setup.py

```diff
@@ -0,0 +1,59 @@
+import io
+import os
+from setuptools import setup, find_packages
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+
+def load_readme():
+    with io.open(os.path.join(HERE, "README.rst"), "rt", encoding="utf8") as f:
+        return f.read()
+
+
+def load_about():
+    about = {}
+    with io.open(
+        os.path.join(HERE, "blockstore_config_filesystem", "__about__.py"),
+        "rt",
+        encoding="utf-8",
+    ) as f:
+        exec(f.read(), about)  # pylint: disable=exec-used
+    return about
+
+
+ABOUT = load_about()
+
+
+setup(
+    name="tutor-contrib-blockstore-filesystem",
+    version=ABOUT["__version__"],
+    url="https://github.com/brian-smith-tcril/tutor-contrib-blockstore-filesystem",
+    project_urls={
+        "Code": "https://github.com/brian-smith-tcril/tutor-contrib-blockstore-filesystem",
+        "Issue tracker": "https://github.com/brian-smith-tcril/tutor-contrib-blockstore-filesystem/issues",
+    },
+    license="AGPLv3",
+    author="Brian Smith",
+    description="blockstore FileSystemStorage configuration plugin for Tutor",
+    long_description=load_readme(),
+    packages=find_packages(exclude=["tests*"]),
+    include_package_data=True,
+    python_requires=">=3.7",
+    install_requires=["tutor"],
+    entry_points={
+        "tutor.plugin.v1": [
+            "blockstore_config_filesystem = blockstore_config_filesystem.plugin"
```

Comment:
```suggestion
            "blockstore-filesystem = blockstore_config_filesystem.plugin"
```

I figure it's more natural for the entry point name - effectively the plugin name - to use dashes instead of underscores.  Plus, having it be a little shorter can't hurt, as there's no built-in auto-complete for `tutor plugins enable`. :)